pymars/drg.py

```diff
@@ -416,6 +417,32 @@ def get_rates_drg(sim_array, solution_object):
         total_edge_data[ic] = ic_edge_data
     return total_edge_data
 
+def target_error_check(solution, target_species):
+    """
+    check if target species are present in graph before attempting to remove them, 
```

Comment:
This is making sure its in the solution, not in the graph.  Graph hasnt been made yet but if its in the solution it should be added to the graph.